README.md

```diff
@@ -3,12 +3,35 @@
 # Xlang
 
 ## Overview
-Xlang is a system that enables a component written in any supported language to be callable from any other supported language using object-oriented programming constructs.  
+The xlang project enables developers to take existing shared libraries,
+implemented in one programming language,
+and make that library's APIs available to client code using a different programming language.
```

Comment:
This is imprecise since this suggests that existing APIs need no changes. Please reword to clarify that this makes it possible to write APIs that are …, rather than "use what you have"